docs/topics/native/apple-framework.md

```diff
@@ -1,363 +1,329 @@
-the [kotlin-multiplatform](gradle-configure-project.md#targeting-multiple-platforms)
-plugin is used. Use the following block to configure the project
-to generate a dynamic or shared library: 
-
-```kotlin
-binaries {
-  framework {
-    baseName = "Demo"
-  }  
-}
-```
-
-Along with macOS `X64`, Kotlin/Native supports macos `arm64` and iOS `arm32`, `arm64` and `X64`
-targets. You may replace the `macosX64` with respective functions as shown
-in the table:
-
-| Target platform/device | Gradle function     |
-|------------------------|---------------------|
-| macOS x86_64           | `macosX64()`        | 
-| macOS ARM 64           | `macosArm64()`      |
-| iOS ARM 64             | `iosArm64()`        | 
-| iOS Simulator (x86_64) | `iosX64()`          |
-| iOS Simulator (arm64)  | `iosSimulatorArm64` |
-
-Run the `linkNative` Gradle task to build the library 
-[in the IDE](native-get-started.md) 
-or by calling the following console command:
-
-```bash
-./gradlew linkNative
-```
-
-Depending on the variant, the build generates the framework
-into the
-`build/bin/native/debugFramework`
-and
-`build/bin/native/releaseFramework`
-folders.
-Let's see what is inside.
+    
+    kotlin {
+        macosX64("native") {
+            binaries {
+                framework {
+                    baseName = "Demo"
+                }
+            }
+        }
+    }
+    
+    tasks.wrapper {
+        gradleVersion = "%gradleVersion%"
+        distributionType = Wrapper.DistributionType.ALL
+    }
+    ```
+    
+    </tab>
+    <tab title="Groovy" group-key="groovy">
+    
+    ```groovy
+    plugins {
+        id 'org.jetbrains.kotlin.multiplatform' version '%kotlinVersion%'
+    }
+    
+    repositories {
+        mavenCentral()
+    }
+    
+    kotlin {
+        macosX64("native") {
+            binaries {
+                framework {
+                    baseName = "Demo"
+                }
+            }
+        }
+    }
+    
+    wrapper {
+        gradleVersion = "%gradleVersion%"
+        distributionType = "ALL"
+    }
+    ```
+    
+    </tab>
+    </tabs>
+
+    The `binaries {}` block configures the project to generate a dynamic or shared library.
+
+    Along with `macosX64`, Kotlin/Native supports the `macosArm64` target for macOS and the `iosX64`, `iosArm64`, and
+    `iosSimulatorArm64` targets for iOS. You can replace the `macosX64` with the respective functions:
+    
+    | Target platform/device | Gradle function       |
+    |------------------------|-----------------------|
+    | macOS x86_64           | `macosX64()`          | 
+    | macOS ARM 64           | `macosArm64()`        |
+    | iOS ARM 64             | `iosArm64()`          | 
+    | iOS x86_64             | `iosX64()`            |
+    | iOS Simulator (arm64)  | `iosSimulatorArm64()` |
+
+3. Run the `linkNative` Gradle task to build the library in the IDE or use the following call in the command line:
```

Comment:
Yeah, it works. You can see it in the list of Gradle build tasks. there are actually 4, `linkNative`, `linkDebugFrameworkNative`,  `linkReleaseFrameworkNative`,`linkDebugTestNative`

Reply:
Oh. Does running `linkNative` compile all the binaries then? It would take a lot of time, so might be annoying.

Reply:
it compiles both debug- and release frameworks, should we use just `linkReleaseFrameworkNative` in this tutorial instead to create only the release variant?

Reply:
> should we use just `linkReleaseFrameworkNative` in this tutorial instead to create only the release variant?

I think we should use `linkDebugFrameworkNative`, since it is much faster.